recipes/bioconductor-bayespeak/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: bioconductor-bayespeak
+  version: 1.24.0
+source:
+  fn: BayesPeak_1.24.0.tar.gz
+  url: https://bioarchive.galaxyproject.org/BayesPeak_1.24.0.tar.gz
+  md5: 821ed8cbee83a5b8dfd7530306d9e3ee
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - bioconductor-iranges
+    - 'r >=2.14'
```

Comment:
That was just what the `bioconductor_skeleton.py` script generated.  I can change it to `r-base` if that's the preferred convention.